src/Pages/HumansAtNasa/HumansAtNasa.tsx

```diff
@@ -0,0 +1,18 @@
+﻿﻿import React from "react";
+import {NavBar} from "../../NavBar/NavBar";
+import {HumanAtNasa} from "../../Components/HumanAtNasa/HumanAtNasa";
+import './HumansAtNasa.scss';
+
+export function HumansAtNasa() {
+    return (
+        <div className='humansAtNasaBody'>
+            <NavBar/>
```

Comment:
We shouldn't need this nav bar - should already be added by the `App` component.

Reply:
deleted.